docs/mkdocs/docs/technical/releasing.md

```diff
@@ -70,9 +70,15 @@ You will need to update:
 A PR is generally open with a todo-list summarizing all the required steps to perform,
 before an update to the feedstock.
 
+If the bot does not create a PR, you can create one yourself by following these steps:
+
+1. If fork not already created, navigate to https://github.com/conda-forge/arcticdb-feedstock and click Fork -> Create Fork
+2. On your repo, create a new branch for your change: `Branches`, `New Branch`, `conda-forge/arcticdb-feedstock`, `rc` or `main`
```

Comment:
Nitpick: a verbose alternative.
```suggestion
2. On your repo, create a new branch for your change. This can be done by selecting
`Branches` and then `New Branch`. Use the `rc` branch or the `main` branch of
`conda-forge/arcticdb-feedstock` as the base branch.
```